.github/workflows/build.yml

```diff
@@ -25,6 +25,40 @@ jobs:
       - name: create bundle
         run: yarn package
 
+      - name: Upload artifact
+        uses: actions/upload-artifact@v1
+        with:
+          name: dist
+          path: dist
+
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-latest, macos-latest, ubuntu-latest]
+    needs: build
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+
+      - name: Install pipenv
+        run: pip install pipenv
+
+      - name: download artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: dist
+          path: dist
+
+      - name: extract dist
+        run: yarn extract
+        shell: bash
+
       - name: integration tests
         run: yarn integ
-
+        shell: bash
```

Comment:
what does that do?

Reply:
The default shell for non-Windows is `bash` and for Windows is `powershell`. Explicitly setting this so that the Windows build uses bash. Here is the `bash` info for Github Actions:

```
The default shell on non-Windows platforms with a fallback to sh.
When specifying a bash shell on Windows, the bash shell included with Git for Windows is used.
```


Docs: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell